Update Code - Fix play URI method.

tenmaster opened the pull request:
- Fixes an error in the example, that prevented URI replay.
- Code refinement

Commit:
Update Code - Fix play URI method.

ramabit commented on justplay/src/main/java/inaka/justplay/core/JustPlay.java:
```diff
@@ -8,17 +8,21 @@
 
 /**
- * Created by emi on 1/8/16.
+ * Created by tenmaster on 1/8/16.
```
According to our Android internal conventions, we prefer not to add headers indicating the creator/owner of the file

ramabit commented:
@tenmaster it is necessary to upload the justplay-justplay.iml file or it can be added to the .gitignore?

Commit:
Suggestion made by Fer

Commit:
Update README

Commit:
Remove unnecessary library

Commit:
Rollback the gitignore

Isn't necessary to update the project iml, but having it in the repo
makes it easier to compile a fresh clone.
